istioctl/pkg/util/handlers/handlers_test.go

```diff
@@ -44,6 +44,18 @@ func TestInferPodInfo(t *testing.T) {
 			wantPodName:   "istio-ingressgateway-8d9697654-qdzgh",
 			wantNamespace: "kube-system",
 		},
+		{
+			proxyName:     "istio-security-post-install-1.2.2-bm9w2.istio-system",
```

Comment:
We never wanted to support names like `foo.ns.svc.cluster.local` right?

Reply:
This is used to parse pod name, the `foo.ns.svc.cluster.local` is for svc format.